src/pages/library/dataBrowser-utils.js

```diff
@@ -123,14 +132,34 @@ export const useDataCatalog = () => {
   return { dataCatalog, refresh, loading }
 }
 
-export const importDataToWorkspace = datasets => {
-  // TODO (DC-284): Call data catalog to figure out what the format should be for importing to workspace
-  const format = 'snapshot'
-  Nav.history.push({
-    pathname: Nav.getPath('import-data'),
-    search: qs.stringify({
-      url: getConfig().dataRepoUrlRoot, format, referrer: 'data-catalog',
-      snapshotIds: _.map('dct:identifier', datasets)
-    })
-  })
+export const importDataToWorkspace = async (dataset, asyncHandler) => {
+  const routeOptions = await Utils.cond(
+    [isWorkspace(dataset), () => {
+      return {
```

Comment:
Nit: This `return` wrapping could be removed for concision as explained in an earlier comment.